src/System.Net.Sockets/tests/FunctionalTests/SocketAPMExtensions.cs

```diff
@@ -29,12 +29,17 @@ public static void ReceiveAPM(this Socket socket, byte[] buffer, int offset, int
             socket.BeginReceive(buffer, offset, count, flags, callback, socket);
         }
 
-        public static void SendFileAPM(this Socket socket, string filename, byte[] preBuffer, byte[] postBuffer, TransmitFileOptions flags, Action handler)
+        public static void SendFileAPM(this Socket socket, string filename, byte[] preBuffer, byte[] postBuffer, TransmitFileOptions flags, Action<Exception> handler)
```

Comment:
Adding the Exception arg is definitely an improvement.  That said, none of the other APM wrappers here have it.  Should we at least file an issue to get those changed as well?